skins/laika/src/components/pages/donation_form/subpages/PaymentPage.vue

```diff
@@ -4,7 +4,8 @@
 		class="payment-page"
 		ref="payment"
 		action="/donation/add"
-		method="post">
+		method="post"
+		@keydown.enter.prevent="next()">
```

Comment:
This will call the `next()` method too early, leading to error messages and not being able to "escape" fields via "enter" (enter acting more like "Tab"). I'd prefer a behavior where hitting enter blurs the fields (and thus triggers the blur behavior). Please consult with Jan for the behavior.

Reply:
After checking some more, I saw that I ran into Firefox-on-Mac bugs/features. See https://bugzilla.mozilla.org/show_bug.cgi?id=756028 and https://bugzilla.mozilla.org/show_bug.cgi?id=1535742